EIPS/eip-894.md

```diff
@@ -0,0 +1,36 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for any state changes to the Ethereum blockchain should not be merged into the official Ethereum repositories.
+
+
+## Motivation
+The issue of bailouts on the Ethereum blockchain is always controversial. This EIP attempts to raise the barriers for bailouts.
+
+
+## Specification
+If a pull request is opened that advocates for bailouts or implements code that can result in state changes on the Ethereum blockchain, it will be closed.
+
+
+### Justification
+Bailouts play favorites with some accounts on the Ethereum blockchain. This is corruption we can not tolerate in the offical Ethereum repositories.
+
+
+## Rationale
+The primary consideration for the approach described above was to reject all bailouts.  A secondary consideration was to standardize the response to such pull requests.
```

Comment:
I fail to find the actual response in this proposal.

Reply:
The response is to close down such pull requests.